client/my-sites/domains/domain-management/list/domains-table-header.jsx

```diff
@@ -56,7 +56,7 @@ class DomainsTableHeader extends PureComponent {
 	};
 
 	renderSortIcon( column, sortKey, sortOrder ) {
-		if ( ! column?.isSortable ) {
+		if ( ! column?.isSortable || ! column?.supportsOrderSwitching ) {
```

Comment:
@candy02058912 I don't think this change is still needed.

Reply:
Yup we don't need it.